main.js

```diff
@@ -93,7 +93,10 @@ const askForSecrets = async () => {
 	if (config.get("discordBot")) {
```

Comment:
```suggestion
        config.discordBot = config.BotToken !== "" ? false : config.get("discordBot");
	if (config.get("discordBot")) {
```

Reply:
is that all that needs to be added, or is there stuff that has to be re-worded/added?

Reply:
That should do it

Reply:
ok thank you, I will add that line then test